### PR TITLE
Added popup window

### PR DESCRIPTION
**Pull request by nahnahdev**
## In This Branch

- [ ] When a button is clicked, the popup with details about the project appears.
- [ ] When the close (X) button is clicked, the popup disappears.
- [ ] Data was stored and accessed from a Javascript object to load Data dynamically.
- [ ] After that, you could implement the popup window.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -424,5 +424,6 @@ <h4 class="desktop-header">Get started</h4>
       </div>
     </footer>
     <script src="menu.js"></script>
+    <script src="popup.js"></script>
```
Comment (Ismailco) [inline index.html:289]: - [ ] The projects section should be created dynamically using JavaScript, kindly consider removing the project section from your HTML file from around line `100` to line `289`, and generate this section (Project section) dynamically on your script file

Comment (Ismailco) [inline index.html:289]: - [ ] The mobile popup on the mobile view has more space between the buttons and the paragraph, kindly consider fixing this so your design looks more professional and fits all screen sizes properly

Screenshot_20220908_194202

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @NahnahAJ, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @NahnahAJ, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**